ip/manager.go

```diff
@@ -126,3 +127,11 @@ func (m *manager) sendEvent(status string) {
 	}
 	m.eventChan <- status
 }
+
+func (m *manager) SetHealthchecks(ctx context.Context, cfg config.Config, healthchecks []models.Healthcheck) {
+	log := logger.Get(ctx)
+	log.WithField("healtchchecks", healthchecks).Debug("Set new healthchecks")
+
+	m.ip.Checks = healthchecks
+	m.checker = healthcheck.FromChecks(cfg, healthchecks)
```

Comment:
This should be protected by a mutex since we might be running the checks when a user asks for an update.

Reply:
From what I understand, this is not really an issue if writing to `checker` is not protected by a mutex. What is the risk here? One health check might fail if it is executed in the middle of the update of `checker`? This is not really a problem, don't you think?

Reply:
Yes it might be an issue because the healthcheck it might make the coroutine panic.